### DISCUSSION TITLE
Control two inputs with one register function

### DISCUSSION THREAD

**Discussion by alexandrzavalii** [react-hook-form/react-hook-form#7114] [Q&A]
I have two inputs which represent the same value

```
<input type="number" />
<input type="range"/>
```
I need a way to control them with single register, for performance reasons. 

I tried here with a hack to set value to the input type number when the range is changing, but I am struggling to make the vice-verse connection.

https://codesandbox.io/s/hopeful-ptolemy-nqfgr?file=/src/App.js

Is this something that was considered before?

**Comment by bluebill1049** [comment]
consider to build this into a controlled component and wrapped with controller.
